app/components/Channels/ChannelSummaryListItem.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Flex } from 'rebass'
+import { Card, Heading, Text } from 'components/UI'
+import { ChannelCapacity, ChannelMoreButton, ChannelStatus } from 'components/Channels'
+import withEllipsis from 'components/withEllipsis'
+
+const ClippedHeading = withEllipsis(Heading.h1)
+const ClippedText = withEllipsis(Text)
+
+const ChannelSummaryListItem = ({
+  channelId,
+  channelName,
+  channelPubKey,
+  cryptoCurrency,
+  cryptoCurrencies,
+  localBalance,
+  remoteBalance,
+  status,
+  setCryptoCurrency,
+  showChannelDetail,
+  ...rest
+}) => {
+  return (
```

Comment:
can remove the return here and make this a pure component

Reply:
Updated.